concrete-commons/src/parameters.rs

```diff
@@ -12,6 +16,30 @@ pub struct CleartextCount(pub usize);
 #[derive(Copy, Clone, Eq, PartialEq, Debug, Serialize, Deserialize)]
 pub struct CiphertextCount(pub usize);
 
+/// The number of ciphertexts in an lwe ciphertext list.
+#[derive(Copy, Clone, Eq, PartialEq, Debug, Serialize, Deserialize)]
+pub struct LweCiphertextCount(pub usize);
+
+/// The index of a ciphertext in an lwe ciphertext list.
+#[derive(Copy, Clone, Eq, PartialEq, Debug, Serialize, Deserialize)]
+pub struct LweCiphertextIndex(pub usize);
+
+/// The range of indices of multiple contiguous ciphertexts in an lwe ciphertext list.
+#[derive(Copy, Clone, Eq, PartialEq, Debug, Serialize, Deserialize)]
+pub struct LweCiphertextRange(pub usize, pub usize);
+
+/// The number of ciphertexts in a glwe ciphertext list.
+#[derive(Copy, Clone, Eq, PartialEq, Debug, Serialize, Deserialize)]
+pub struct GlweCiphertextCount(pub usize);
+
+/// The number of ciphertexts in a gsw ciphertext list.
+#[derive(Copy, Clone, Eq, PartialEq, Debug, Serialize, Deserialize)]
+pub struct GswCiphertextCount(pub usize);
+
+/// The number of ciphertexts in a gsw ciphertext list.
+#[derive(Copy, Clone, Eq, PartialEq, Debug, Serialize, Deserialize)]
+pub struct GgswCiphertextCount(pub usize);
+
 /// The number of scalar in an LWE mask + 1 .
```

Comment:
Not sure I understood this remark !

Reply:
I assume you speak about the first one. What I meant is that it might not be clear for a non-expert why it is useful to have an entire struct dedicated to a simple addition by one (and why not by two for example?). So maybe we could change that comment by:
/// The number of scalar in an LWE ciphertext, i.e. the number of scalar in an LWE mask plus one.

Reply:
Also, shouldn't it be "scalarS"?

Reply:
I agree with everything ! Fixed ! Thanks !